test.py

```diff
@@ -0,0 +1,94 @@
+#fasta
+fasta = open("S_cerevisiae/saccharomyces_cerevisiae_R64-1-1_20110208_sequences.fasta", "r")
+
+seq = fasta.read()
+seqlist = seq.split("\n")
+seqlist_new = []
+name = []
+count = -1
+
+for i in range(len(seqlist)):
+    if ">" in seqlist[i]:
+        name.append(seqlist[i][1:])
+        count += 1
+        seqlist_new.append("")
+    else:
+        seqlist_new[count] += seqlist[i]
```

Comment:
`i`が`seqlist[i]`としてしか使われていない。わざわざ`i`を使うとバグりやすいし管理しにくい。そのまま中身を使うと楽
```suggestion
for seq in seqlist:
    if ">" in seq:
        name.append(seq[1:])
        count += 1
        seqlist_new.append("")
    else:
        seqlist_new[count] += seq
```